python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -239,6 +251,27 @@ def __volumeSummary( plug ) :
 
 		],
 
+		# Transform
+	
+		"attributes.transformType" : [
+
+			"description",
+			"""
+			Choose how transform motion is interpolated.  Linear produces classic Renderman-style linear vertex motion, rotate_about_origin produces curves arcs centerd on the object's origin, and rotate_about_center, the default, produces curved arcs centred on the object's bounding box middle.
+			""",
+
+			"layout:section", "Transform",
+
+		],
+		"attributes.transformType.value" : [
+
+			"presetNames", lambda plug : IECore.StringVectorData( __transformTypeEnumNames ),
```

Comment:
Quick comment for when we switch to string values. Elsewhere we've used Gaffer-style CamelCase for the enum names and then "arnold_style" for the values. Let's do the same here.